new-lamassu-admin/src/pages/Locales/Locales.js

```diff
@@ -121,6 +140,13 @@ const Locales = ({ name: SCREEN_KEY }) => {
     return saveConfig({ variables: { config } })
   }
 
+  const enableCoin = it => {
+    if (!it) return
+
+    const namespaced = fromNamespace(it)(wallets)
+    if (!namespaced?.active) return setWizard(it)
```

Comment:
I don't get this one. If we're removing the active prop when would this ever be true? Don't we want to check if coin is configured here?

Reply:
Still here.